src/pages/en/reports/accessibility/2022-07.md

```diff
@@ -0,0 +1,213 @@
+
+So congratulations to
+
+- [Email On Acid](https://www.emailonacid.com/) (1 email)
+- [Blocks Edit](https://blocksedit.com/) (2 emails)
+- [Basta!](https://basta.media/) (8 emails)
+
+All three of these brands are setting a great example of what can be done.
+
+After some additional manual testing on these campaigns, the Email On Acid email came out on top. So if you’re looking for an example of what to aim for, you can [view the most accessible email from our test](https://marketing.emailonacid.com/ebook-the-omnichannel-heros-journey-2).
+
+### Top 10 most common accessibility issues in email
+
+#### (1) Serious: Content inside the body should be wrapped in a `lang` attribute
+
+Found in 86.1% of emails tested.
+
+#### Why it’s needed:
+
+This defines the language for the email, it can help with translation tools but the main reason it’s needed is for tools like screen readers. If a screen reader is set to the wrong language it can change the pronunciation of words which in turn could make it completely unusable.
+
+#### How to fix this:
+
+This can be fixed by adding a `<div>` as the first child of the `<body>` element with a `lang=""` attribute set on it with the appropriate language. We need this set inside the `<body>` and on the `<html>` tag as some email clients will strip parts of the code before displaying it.
+
+When setting a `lang=""` attribute it’s always good to set a `dir=""` attribute too, to define the direction of the language.
+Learn more about `lang` attributes
+
+---
+
+#### (2) Moderate: Page should contain a level-one heading
+
+Found in 85.7% of emails tested.
+
+##### Why it’s needed:
+
+According to the [2021 WebAIM screen reader user survey](https://webaim.org/projects/screenreadersurvey9/#finding) 67.7% use headings as their primary way of finding information. Having an `<h1>` in the email means they can quickly and easily find the email in the page and the information inside it.
+
+##### How to fix this:
+
+Add an `<h1>` tag around the main heading of the email. You should only use one `<h1>` after that you can use `<h2>` headings as sub headings under that main heading, and `<h3>` as a sub heading of the `<h2>` all the way down to `<h6>`.
+
+---
+
+#### (3) Serious: `<html>` element must have a `lang` attribute
+
+Found in 78.8% of emails tested.
+
+##### Why it’s needed:
+
+This defines the language for the email, it can help with translation tools but the main reason it’s needed is for tools like screen readers. If a screen reader is set to the wrong language it can change the pronunciation of words which in turn could make it completely unusable.
+
+##### How to fix this:
+
+This can be fixed by adding a `lang=""` attribute to the `<html>` tag. This is in addition to the [Content inside the body should be wrapped in a `lang` attribute](https://docs.google.com/document/d/16M3gFklJt_z_GbeCMWXxMevJRrF9WsrjMCnAY39HK9I/edit#heading=h.7a1a3e736rfw) issue mentioned above.
```

Comment:
update link to, inside this document
#1-serious-content-inside-the-body-should-be-wrapped-in-a-lang-attribute

Reply:
Good catch